helpers/integration-tests_deprecated.js

```diff
@@ -0,0 +1,53 @@
+const puppeteer = require('puppeteer');
```

Comment:
We no longer need this file 🙂

Reply:
Oh, I forgot that one! Thanks!

Reply:
It looks like this file is still in the PR